CHANGELOG.md

```diff
@@ -13,7 +13,13 @@ The changes are relative to the previous release, unless the baseline is specifi
 * Set avifDecoder::image->depth to the same value after avifDecoderParse() as
   after avifDecoderNextImage() when AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM is
   enabled and when the file to decode contains a 'sato' derived image item.
-* Update rav1e.cmd/LocalRav1e.cmake: cargo-c v0.10.13, corrosion v0.5.2, rav1e v0.8.0
+* Update rav1e.cmd/LocalRav1e.cmake: cargo-c v0.10.13, corrosion v0.5.2,
+  rav1e v0.8.0
+
+### Removed since 1.3.0
+
+* ext/avm.cmd was removed. Use the -DAVIF_CODEC_AVM=LOCAL CMake build option
+  instead.
```

Comment:
The second sentence, "Use the -DAVIF_CODEC_AVM=LOCAL CMake build option instead.", is not accurate. `-DAVIF_CODEC_AVM=LOCAL` means "use a local libavm build", and ext/avm.cmd (if it works) is one of two ways to use a local libavm build.

If `-DAVIF_CODEC_AVM=LOCAL` is specified, we first see if there is a libavm build in the ext/avm directory (which we assume was created by running ext/avm.cmd). If Yes, we are done. If No, then we run cmake/Modules/LocalAvm.cmake to fetch and build libavm.